tests/cpp/fwklib/FwkLog.cpp

```diff
@@ -54,32 +55,24 @@ const char* dirAndFile(const char* str) {
 }
 
 void plog(const char* l, const char* s, const char* filename, int32_t lineno) {
```

Comment:
If it simlifies things, the only callers to this method are macros that are converting a `std::ostringstream` to a `char *` so you could just pass a `std::ostringstream &` or `const std::string &` here.

```c++
void plog(const std::string& level,  const std::string& message, const std::string& filename, size_t lineNumber);
```